examples/dao/setup_sigma_dao.sql

```diff
@@ -11,8 +11,12 @@ CREATE TABLE IF NOT EXISTS sigma_daos (
   asset_id BIGINT -- token id
 );
 
--- Create unique index on app_id
+-- Create indexes
 CREATE UNIQUE INDEX sigma_daos_app_id_idx ON sigma_daos (app_id);
+-- create an index to do efficient text search using `%<text>%`, it requires
+-- pg_trgm extension.
+CREATE EXTENSION pg_trgm;
+CREATE INDEX sigma_daos_dao_name_idx ON users USING gin (dao_name gin_trgm_ops);
```

Comment:
Are there any tests/performance benchmarks for the querying we're doing using postgres?

Reply:
no, we don't have a test data. Would be good to generate some. But probably later once we are closer to production.